dl/src/main/scala/com/intel/analytics/bigdl/optim/SGD.scala

```diff
@@ -171,12 +196,12 @@ object SGD {
     }
   }
 
-  case class Default() extends LearningRateSchedule {
+  case class Default() extends HyperParameterScheduler {
     override def updateHyperParameter(config: Table, state: Table): Unit = {
       val lr = config.get[Double]("learningRate").getOrElse(1e-3)
       val lrd = config.get[Double]("learningRateDecay").getOrElse(0.0)
       val nevals = state.get[Int]("evalCounter").getOrElse(0)
-      config("clr") = -lr / (1 + nevals * lrd)
+      config("clr") = lr / (1 + nevals * lrd)
```

Comment:
why change this? If you want to display a positive learningRate, you should negative it at the display part.